configs/icds-14lakh.yml

```diff
@@ -0,0 +1,563 @@
+  sms_monthly_total:
+    model: 'derived_sum'
+    start_with: 3438200
+    dependant_fields:
+      - 'sms_adolescent_girls'
+      - 'sms_pregnancies'
+      - 'sms_lactating_women'
+      - 'sms_aww'
+      - 'sms_ls'
+  sms_total:
+    model: 'cumulative'
+    dependant_field: 'sms_monthly_total'
+  sms_models_total:
+    model: 'derived_factor'
+    dependant_field: 'sms_total'
+    factor: 10  # sms (1) + smsbillables (1) + messagingevent (4) + messagingsubevent (4)
+
+  kafak_changes:
+    # we only keep kafka changes for 28 days so no need to calculate the cumulative amount
+    model: 'derived_sum'
+    dependant_fields:
+      - 'forms'
+      - 'cases'
+      - 'case_transactions'
+      - 'ledger_updates_per_month'
+      - 'synclogs'
+      - 'sms_monthly_total'
+
+services:
+  pg_shards:
+    # need higher performing SSDs
+    usage_capacity_per_node: 50000
+    storage:
+      group: 'SSD'
+      data_models:
+        - referenced_field: 'forms_total'
+          unit_size: 1200
+        - referenced_field: 'cases_total'
+          unit_size: 1800
+        - referenced_field: 'case_indices'
+          unit_size: 380
+        - referenced_field: 'case_transactions_total'
+          unit_size: 515
+        - referenced_field: 'ledgers'
+          unit_size: 450
+        - referenced_field: 'ledger_updates'
+          unit_size: 420
+        - referenced_field: 'images_total'
+          unit_size: 370
+        - referenced_field: 'sms_total'  # TODO should be modeled separately from SMS since it's SMS triggers
+          unit_size: 683  # casetimedscheduleinstance
+    process:
+      cores_per_node: 32
+      ram_per_node: 128
+
+  pg_proxy:
+    usage_capacity_per_node: 300000
+    storage_scales_with_nodes: True
+    storage:
+      group: 'VM_other'
+      static_baseline: 100GB
+      add_buffer: False
+      override_estimation_buffer: 0
+    process:
+      cores_per_node: 16
+      ram_per_node: 64
+
+  pg_main:
+    usage_capacity_per_node: 250000
+    storage_scales_with_nodes: True
+    min_nodes: 2
+    storage:
+      group: 'SSD'
+      static_baseline: 50GB  # to account for other static tables
+      data_models:
+        - referenced_field: 'phone_numbers'
+          unit_size: 550
+        - referenced_field: 'users'
+          unit_size: 1600
+        - referenced_field: 'async_indicators'
+          unit_size: 6626
+        - referenced_field: 'sms_models_total'
+          unit_size: 510  # averaged over all models
+    process:
+      cores_per_node: 32
+      ram_per_node: 128
+
+  pg_synclogs:
+    usage_capacity_per_node: 500000
+    storage:
+      group: 'SSD'
+      data_models:
+        - referenced_field: 'synclogs'
+          unit_size: 210000
+    process:
+      cores_per_node: 32
+      ram_per_node: 128
+
+  couchdb:
+    usage_capacity_per_node: 100000
+    min_nodes: 3
+    storage:
+      group: 'SSD'
+      redundancy_factor: 3
+      static_baseline: 20GB  # to account for other databases
```

Comment:
Does this also account for the need for extra disk space when compacting?

Reply:
Probably not, thanks for flagging. I'll override the buffer for couch to make it bigger. Any thoughts on how how much to account for (%)?

Reply:
Needs free space equal to the size of the view / DB being compacted